src/commands/up.rs

```diff
@@ -127,10 +135,25 @@ impl UpCommand {
                     .as_deref()
                     .unwrap_or_else(|| DEFAULT_MANIFEST_FILE.as_ref());
                 let bindle_connection = self.bindle_connection();
-                spin_loader::from_file(manifest_file, &working_dir, &bindle_connection).await?
+                spin_loader::from_file(
+                    manifest_file,
+                    if self.direct_mounts {
```

Comment:
I found the inlining here really threw off my reading - this is a simple function call with three arguments, but the inlining spreads it over 10 lines and three levels of indentation.  Can we not create a variable (e.g. `asset_dest`) on the line before and replace `&working_dir` in the original call with that?